src/rabbit.erl

```diff
@@ -848,6 +852,82 @@ warn_if_kernel_config_dubious() ->
         true  -> ok
     end.
 
+warn_if_disc_io_options_dubious() ->
+    %% if these values are not set, it doesn't matter since
+    %% rabbit_variable_queue will pick up the values defined in the
+    %% IO_BATCH_SIZE and CREDIT_DISC_BOUND constants.
+    CreditDiscBound = rabbit_misc:get_env(rabbit, credit_disc_bound, undefined),
```

Comment:
Please prefix these with `msg_store_`, e.g. we already have `msg_store_file_size_limit`, `queue_index_embed_msgs_below`, and `queue_index_max_journal_entries`.